src/Http/Routing/test/UnitTests/EndpointRoutingMiddlewareTest.cs

```diff
@@ -159,19 +172,46 @@ private HttpContext CreateHttpContext()
         private EndpointRoutingMiddleware CreateMiddleware(
             Logger<EndpointRoutingMiddleware> logger = null,
             MatcherFactory matcherFactory = null,
+            DiagnosticListener listener = null,
             RequestDelegate next = null)
         {
             next ??= c => Task.CompletedTask;
             logger ??= new Logger<EndpointRoutingMiddleware>(NullLoggerFactory.Instance);
             matcherFactory ??= new TestMatcherFactory(true);
+            listener ??= new DiagnosticListener("Microsoft.AspNet");
```

Comment:
nit: `Microsoft.AspNetCore` is the name at runtime.
```suggestion
            listener ??= new DiagnosticListener("Microsoft.AspNetCore");
```